lib/cousins.ml

```diff
@@ -301,14 +301,44 @@ let cousins_fold l =
         loop false
           (if first || cnt0 = 0 then acc
           else (ip0, (ifaml0, iancl0, cnt0), lev0) :: acc)
-          (ip, (ifaml, [ ianc ], 1), lev)
+          (ip, ([ ifaml ], [ ianc ], 1), lev)
           l
     | [] ->
         if first || cnt0 = 0 then acc
         else (ip0, (ifaml0, iancl0, cnt0), lev0) :: acc
   in
   loop false [] (Gwdb.dummy_iper, ([], [], 0), [ 0 ]) l
 
+let cousins_implex_cnt base max_a_l max_d_l l1 l2 p =
+  let il1 = int_of_string l1 in
+  let il2 = int_of_string l2 in
```

Comment:
I would prefer to do this conversion before `cousins_implex_cnt`, to have `val cousins_implex_cnt : base -> int -> int -> int -> int -> person -> int`

Reply:
If we do it, the same comment applies to cousins_l1_l2_aux?